src/node/structural_equal.cc

```diff
@@ -189,16 +201,19 @@ class RemapVarSEqualHandler :
     bool map_free_vars;
     /*! \brief Whether the children has been expanded via SEqualReduce */
     bool children_expanded{false};
+    /*! \brief whether the task is about graph equality(need remap). */
+    bool graph_equal{false};
 
     Task() = default;
     Task(ObjectRef lhs, ObjectRef rhs, bool map_free_vars)
         : lhs(lhs), rhs(rhs), map_free_vars(map_free_vars) {}
   };
-
   // list of pending tasks to be pushed to the stack.
   std::vector<Task> pending_tasks_;
   // Internal task stack to executed the task
   std::vector<Task> task_stack_;
+  // record current stack top
```

Comment:
should we also use /*! \brief */ comment style here?

Reply:
Given that this is private and won't generate docs, perhaps we can just go with this way